library/src/main/java/com/getstream/sdk/chat/model/Channel.java

```diff
@@ -943,24 +943,20 @@ public void show(@NotNull CompletableCallback callback) {
     /**
      * edit the channel's custom properties.
      *
-     * @param options       the custom properties
      * @param updateMessage message allowing you to show a system message in the Channel that something changed
      * @param callback      the result callback
      */
-    public void update(@NotNull Map<String, Object> options, @Nullable String updateMessage,
-                       @NotNull ChannelCallback callback) {
-        client.updateChannel(this, options, updateMessage, callback);
+    public void update(@Nullable String updateMessage, @NotNull ChannelCallback callback) {
```

Comment:
let's change updateMessage from `String` to `Message

Reply:
Should I change ```acceptInvite(@Nullable String message, @NotNull ChannelCallback callback)``` as well?